### ISSUE TITLE
 Separate Data visualisation and calibration from Wax Module

### ISSUE THREAD

**Issue by 12deadpixels** [bcaramiaux/Gestural-Sound-Toolkit#3]
It will be great to separate the Data visualisation module from the Wax accelerometers' one in the Receiver modules.

Does it make sense?

**Comment by bcaramiaux** [comment]
Yes it would make definitely sense. We could have a receiver for the data and a visualiser, using multisliders for instance (other options are MuBu or jitter)

**Comment by 12deadpixels** [comment]
Yes, especially if the data visualisation module is agnostic.
